media/js/mozorg/contribute/friends.js

```diff
@@ -10,4 +19,35 @@ $(function() {
             return $(this).attr('data-png');
         });
     }
-});
+
+    $ffShowSignupForm.on('click', function() {
+        $ffShowSignupForm.addClass('invisible').off('click');
+
+        // slide the form container down
+        $ffSignupArea.slideDown('normal', function() {
+            // scroll to top of revealed form
+            Mozilla.smoothScroll({
+                top: $ffSignupArea.offset().top - 60
+            });
+
+            $('#id_email').focus();
```

Comment:
Hm, this is odd. It only seems to prevent scrolling in Dev Edition and Nightly. Release and Beta work as expected. Simply calling `focus()` before `smoothScroll` seems to fix things, but I wonder if I should file a bug...